content/webapp/components/NewsletterSignup/NewsletterSignup.tsx

```diff
@@ -160,18 +160,6 @@ const NewsletterSignup: FunctionComponent<Props> = ({
             />
           </Space>
 
-          <Space v={{ size: 'xl', properties: ['margin-bottom'] }}>
-            <CheckboxRadio
-              id="whats_on"
-              type="checkbox"
-              text="I'd like to receive regular updates from Wellcome Collection"
-              value="addressbook_40131"
-              name="addressbook_40131"
-              checked={checkedInputs.includes('whats_on')}
```

Comment:
We spoke.

This isn't really a 'double opt-in' checkbox, but a checkbox to receive the 'What's on' newsletter (which is the main one that _most_ people will want to receive).

We _could_ have it as a pre-checked hidden checkbox, but then you wouldn't be able to e.g. only sign up for the 'Access events tours and activities' newsletter.

Think it probs needs a ux/copy rethink

Reply:
As discussed with Jenn, we'll just make sure it copies the behaviour from `NewsletterPromo`, where there is no checkbox and the user is subscribed to What's on on submit.